ocaml/message-switch/core_test/basic-rpc-test.sh

```diff
@@ -3,38 +3,42 @@ set -e
 
 SPATH=${TMPDIR:-/tmp}/sock_s
 SWITCHPATH=${TMPDIR:-/tmp}/switch_s
+SECS=0.1
```

Comment:
so we don't do performance test anymore? Or do we want to split this into basic test, and performance test

Reply:
We never looked at the results of this perf test. We should add it to the stresstest though, and perhaps a 'quickbench', i.e. like 'quicktest' but for benchmarks.

Reply:
I've added back the 5s test as a `dune build @stresstest' alias in commit a6406eaab37e0d4fd2aafa814677a0ba56959e6f